src/neural/blas/network_blas.cc

```diff
@@ -141,12 +142,14 @@ class BlasComputation : public NetworkComputation {
   bool wdl_;
   bool moves_left_;
   bool conv_policy_;
+  bool attn_policy_;
+  bool attn_body_;
+  bool new_encoding_;
```

Comment:
`new_encoding_` is not very descriptive.

Reply:
Renamed.